resources/dicts/events/ceremonies/ceremony_traits.json

```diff
@@ -593,5 +593,23 @@
         "prudence",
         "judgement",
         "foresight"
+    ],
+    "cruel": [
```

Comment:
Did you add a trait? If you wish to propose an additional trait, please remove and PR this separately